src/main/java/uk/gov/companieshouse/accounts/association/repositories/AssociationsRepository.java

```diff
@@ -28,4 +28,8 @@ public interface AssociationsRepository extends MongoRepository<AssociationDao,
     @Query( value = "{ 'company_number': ?0, 'user_id': ?1 }", exists = true )
     boolean associationExists( String companyNumber, String userId );
 
+
+    @Query( "{ 'company_number': ?0, 'user_email': ?1, 'status': { $in: ?2 } } }" )
+    Page<AssociationDao> fetchAssociation( final String companyNumber, final String userEmail, final Set<String> statuses, final Pageable pageable );
```

Comment:
change the name to be more specific based on query.

Reply:
Thanks, I have now renamed the method to be more descriptive